src/compiler/function/FunctionDeclaration.ts

```diff
@@ -79,4 +80,14 @@ export class FunctionDeclaration extends FunctionDeclarationBase<ts.FunctionDecl
     remove() {
         removeOverloadableStatementedNodeChild(this);
     }
+
+    /**
+     * Gets the structure equivalent to this node
+     */
+    getStructure(): FunctionDeclarationStructure {
+        return callBaseGetStructure<FunctionDeclarationSpecificStructure>(FunctionDeclarationBase.prototype, this, {
+            overloads: undefined // TODO: don't know how to implement this
```

Comment:
What about getting the structure from the base and then stripping out any properties that aren't for the overload structure?